### ISSUE TITLE
Fresh build fails due to bad make recipe

### ISSUE THREAD

**Issue by marcopus** [mcastorina/mpv-remote-app#61]
Invoking `make build` after a fresh clone fails with the following error:
```
python3 -m build
/usr/bin/python3: No module named build
make: *** [Makefile:13: dist] Error 1
```
The reason seems to be in the `dist` dependency of the make recipe of `build`:
https://github.com/mcastorina/mpv-remote-app/blob/7e777fd0de228384368def07de2e5a342f863739/server/Makefile#L12-L13

`dist` recipe should probably be something like:
```
$(PYTHON) setup.py sdist bdist_wheel
```

So the `dist` recipe should be fixed, if it serves a different purpose than `build`.

**Comment by marcopus** [comment]
@mcastorina

**Comment by mcastorina** [comment]
Thanks for reporting, I'll take a look soon.

**Comment by mcastorina** [comment]
Fixed in a5f910e4b901ee2c15c1a7a1c368c32c1a343fbb

Not sure what `dist` was doing originally (I was probably getting overly fancy with `make` 🙂)
